### ISSUE TITLE
Interactive Features

### ISSUE THREAD

**Issue by nknapp** [nknapp/bootprint-swagger#29]
This issue is there to collect ideas about interactive features in future bootprint-versions.

**Comment by nknapp** [comment]
* Filtering operations by clicking on tags
* Filtering operations by entering paths into a search-field

**Comment by gnidan** [comment]
This may or may not be interactive, but wondering if it'd be a feasible task to add a swagger-ui style "Model Schema" listing, as this would provide a convenient sample JSON value for a given model.

**Comment by nknapp** [comment]
I'm not sure how this listing is created. Well, both projects are written in javascript, so I guess it must be feasible. I'm not sure if I'll get into it. Could you add this as seperate issue, so it isn't forgotten.

**Comment by gnidan** [comment]
Certainly. Started looking at how this is done in `swagger-ui`, or whether that behavior comes from `swagger-js` or not.

**Comment by nknapp** [comment]
It is now possible to include JavaScript files. The JavaScript files of Bootstrap and jQuery are now included into the output by default.
This means that interactive features can now be implemented.
There is not a lot of documentation yet, but bootprint/bootprint#18 contains some pointers.
